esmvaltool/references/acknow_author.bibtex

```diff
@@ -0,0 +1,3 @@
+@article{acknow_author,
```

Comment:
```suggestion
@misc{acknow_author,
```
The `@article` entry requires at least author, journal, title and year.
So for this case, `@misc` is more appropriate (see [here](https://en.wikibooks.org/wiki/LaTeX/Bibliography_Management#BibTeX)).

Sorry for the step-wise review...